tests/test_login.py

```diff
@@ -0,0 +1,27 @@
+from base_case import BaseTestClass
+
+class TestLogin(BaseTestClass):
+
+    def test_successfull_login(self):
+        email = "arteevraina@gmail.com"
```

Comment:
For each test. It's better to create the user using signup API first. Because we never know which test will run first. So, for example: it can be the case the `test_successful_signup` test does not runs first and `test_successful_login` tests runs first. Then, it might actually fail the test because the user would not be there in the database.

Also, can you fix the spelling of ``successfull`` to ``successful`` in this and other instances as well.

https://dev.to/paurakhsharma/flask-rest-api-part-6-testing-rest-apis-4lla